.github/workflows/xcstrings_conversion.yml

```diff
@@ -6,32 +6,10 @@ on:
       - main
```

Comment:
Could adding a `path` filter here in the workflow trigger suffice for us? Essentially, it looks like we could perhaps filter only for changesets that match something like: `'Scribe-i18n/**.json'` 

[docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-paths)

Haven't tried myself; so not sure. Could be something to attempt though

CC: @Jag-Marcel @andrewtavis

Reply:
Sounds interesting, I think it's worth a try